LayoutTests/fast/events/ios/pdf-modifer-key-down-crash.html

```diff
@@ -7,7 +7,12 @@
 else {
     testRunner.dumpAsText();
     testRunner.queueLoad("../..//images/resources/green_rectangle.pdf");
-    testRunner.queueLoadingScript('setTimeout("history.back()", 0); testRunner.runUIScript(`uiController.keyDown("", ["leftCommand"])`, () => {});');
+    testRunner.queueLoadingScript(`
+        (async function() {
+            setTimeout("history.back()", 0);
+            await testRunner.runUIScript("uiController.keyDown('', ['leftCommand'])", () => {});
```

Comment:
Huh, didn't realize this function started returning a promise.

Reply:
Oh, it didn't (can we return Promises from native code yet?). This patch is bad.

Reply:
We can but I don't think we have hooks wired up in WKTR.

Reply:
@Constellation

Reply:
I think we can use `UIHelper.keyDown` here?